CONTRIBUTING.md

```diff
@@ -66,13 +66,17 @@ Gitpod will automatically install all dependencies; start the webpack server for
 
 The web app doesn't include any performance profiles by default, so you'll need to load some in. Make sure the local Webpack web server is running, and then try one of the following:
 
- * Use an existing profile from the web: replace the `https://profiler.firefox.com` with `http://localhost:4242` (be careful: the leading `https` changes to `http`!).
- * Drag in a saved profile to the loading screen (this makes refreshing hard).
- * Record a new profile.
-  1. Open `about:config` in Firefox.
-  2. Change `devtools.performance.recording.ui-base-url` to `http://localhost:4242` or to the localhost with the proper port you have configured.
-  3. Ensure the profiler menu button is active by clicking the button on the [profiler.firefox.com](https://profiler.firefox.com) homepage to enable it.
-  4. Record a profile using the menu button, and it should open up in your local environment.
+ #### 1. Record a profile: 
+ - Open `about:config` in Firefox.
+ - Change `devtools.performance.recording.ui-base-url` to `http://localhost:4242`. Or to the localhost with the proper port you have configured.
+ - Ensure the profiler menu button is active by clicking the button on the profiler.firefox.com homepage to enable it.
```

Comment:
Previously we had a link on "profiler.firefox.com", can you please add it back?

Reply:
> Thanks, I really like the new structure.
> I gave a few small suggestions to make it even clearer, tell me what you think!

@julienw Thank you, glad to hear that! Changes have been made as requested. Thanks for the feedback. 👍